arch/arm/src/lpc17xx_40xx/lpc17_40_can.c

```diff
@@ -1049,17 +1062,22 @@ static int lpc17can_txpoll(struct net_driver_s *dev)
 
   if (priv->dev.d_len > 0)
     {
-      /* Send the packet */
-
-      lpc17can_transmit(priv);
+      if (!devif_loopback(&priv->dev))
```

Comment:
devif_loopback is specific for ipv4 and ipv6 protocol, is it good to call it in can driver?

Reply:
Thanks for pointing that out, got mixed with backporting this from my older branch.
Should be okay now.